.github/actions/cookiecutter-check/bootstrap.expect

```diff
@@ -0,0 +1,5 @@
+set timeout 180
```

Comment:
If the prompts where to change, then this would timeout and error, yes?

Reply:
Yes it would. We could be more generic in the regex patterns. This felt like an ok balance.

Reply:
Added some pattern matching in bootstrap.expect to give us more leeway in case the cookiecutter output changes in format or something